lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -378,6 +383,9 @@ def setUp(self):
     def test_create_thread(self, mock_request):
         self.create_thread_helper(mock_request)
 
+    def test_create_thread_with_context(self, mock_request):
```

Comment:
`create_thread_helper()` issues a request to the LMS discussion API `create_thread` with some example thread data and `extra_data` if present. It then verifies that the mock comments service was called with the same `extra_data`. All of these tests run against a mock comments service, so there's little to verify beyond checking that the parameters are correctly threaded through the system. See https://github.com/edx/edx-platform/blob/benmcmorran/team-discussion-context/lms/djangoapps/django_comment_client/base/tests.py#L282-L306